eos-core-depends

```diff
@@ -30,7 +30,7 @@ eos-discovery-feed
 eos-event-recorder-daemon
 eos-exploration-center
 eos-gates
-eos-google-chrome-helper [amd64]
+eos-install-app-helper [amd64]
```

Comment:
Drive-by comment: I suppose it is appropriate for now that this is only amd64, but in the long run, I suspect we will have some ARM apps in this state.